### ISSUE TITLE
- pear/log 1.12.8 requires pear/pear_exception 1.0-beta1 -> no matching package found.

### ISSUE THREAD

**Issue by minutephp** [pear/Log#6]
Hi,

I'm trying to install this via composer but it keeps failing with this message:

```
composer require pear/log
Using version ~1.12 for pear/log
./composer.json has been updated
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

 Problem 1
    - Installation request for pear/log ~1.12 -> satisfiable by pear/log[1.12.8].
    - pear/log 1.12.8 requires pear/pear_exception 1.0-beta1 -> no matching package found.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion> for more details.

Read <http://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, reverting ./composer.json to its original content.
```

**Comment by thomasdashney** [comment]
+1

**Comment by tabacco** [comment]
https://github.com/pear/Log/pull/7 should fix that.

**Comment by jparise** [comment]
This has been addressed by #7.
